packages/langium/src/index.ts

```diff
@@ -29,11 +29,12 @@ export * from './parser/langium-parser';
 export * from './references/linker';
 export * from './references/naming';
 export * from './references/scope';
+export * from './test/langium-test';
+export * from './test/parse-helper';
```

Comment:
Please create an `index.ts` in `test` and export the stuff there, so we avoid including the test support in production code. This means you'd import that from `'langium/lib/test'` instead of `'langium'`.